CHANGES.rst

```diff
@@ -1,4 +1,8 @@
-1.0.0 (Unreleased)
+1.1 (Unreleased)
```

Comment:
I think we also want a `1.0.1` entry under here, and would still like sub-divisions for new features, API changes, bug fixes, etc. Just use astropy format...